docs/en/sql-reference/functions/other-functions.md

```diff
@@ -2794,3 +2794,119 @@ message Root
     uint32 column2 = 2;
 }
 ```
+
+## formatQuery
+
+Returns a formatted version of the given SQL query.
+
+**Syntax**
+
+```sql
+formatQuery(query)
+```
+
+**Arguments**
+
+- `query` - The SQL query to be formatted. [String](../../sql-reference/data-types/string.md)
+
+**Returned value**
+
+- The formatted query. [String](../../sql-reference/data-types/string.md).
+
+**Example**
+
+```sql
+SELECT formatQuery('select 1;');
+```
+
+Result:
+
+```result
+┌─formatQuery('select 1;')─┐
+│ SELECT 1                 │
+└──────────────────────────┘
+```
+
+```sql
+SELECT formatQuery('SeLecT 1');
+```
+
+Result:
+
+```result
+┌─formatQuery('SeLecT 1')──┐
+│ SELECT 1                 │
+└──────────────────────────┘
+```
+
+```sql
+SELECT formatQuery('select a,    b FRom tab WHERE a > 3 and  b < 3');
+```
+
+Result:
+
+```result
+┌─formatQuery('select a,    b FRom tab WHERE a > 3 and  b < 3')─┐
+│ SELECT
+    a,
+    b
+FROM tab
+WHERE (a > 3) AND (b < 3)            │
+└───────────────────────────────────────────────────────────────┘
+```
+
+## formatQueryOneLine
+
+Returns a formatted version of the given SQL query on a single line.
```

Comment:
I'd just say `Like formatQuery() but the returned formatted string contains no line breaks.`